pallets/elections/src/traits.rs

```diff
@@ -0,0 +1,76 @@
+use frame_support::{pallet_prelude::Get, traits::Currency};
+use sp_std::vec::Vec;
+
+pub type SessionId = u32;
+pub type EraId = u32;
+
+pub trait SessionInfoProvider<T: frame_system::Config> {
```

Comment:
Could we please add API comments here as well, as those are `pub trait`s

Reply:
added